main.py

```diff
@@ -22,7 +25,7 @@ def fetch_patch():
         'Authorization':f'Bearer {TOKEN}'
         }
     git_request = git_session.get(
-        f'https://api.github.com/repos/{repo}/pulls/{pr}/files',
+        f'https://{api_url}/repos/{repo}/pulls/{pr}/files',
```

Comment:
```suggestion
        f'{api_url}/repos/{repo}/pulls/{pr}/files',
```
I updated the test and it's catching an issue here. I think it's cos `github.api_url` [returns the entire URL](https://github.com/hestonhoffman/changed-lines/actions/runs/10014639165/job/27684696348). I think this will fix it